code/entry.py

```diff
@@ -1,79 +1,98 @@
 import psycopg2
 import easygui
 import json
+import os
+import glob
+
+from code.translate_qep import translate_qeptree_to_text
 
 dbname = 'postgres'
 user = 'postgres'
 host = 'localhost'
 
 connectStatus = False
 
-while(not connectStatus):
-    dbnameInput = input("dbname <"+dbname+">: ")
-    userInput = input("user <"+user+">: ")
-    hostInput = input("host <"+host+">: ")
-    password = input("password: ")
+while not connectStatus:
+    print("*** Enter database credentials to access: ")
+    dbnameInput = input("dbname <" + dbname + ">: ")
+    userInput = input("user <" + user + ">: ")
+    hostInput = input("host <" + host + ">: ")
+    password = easygui.passwordbox("PASSWORD:")
 
-    if (dbnameInput!=''):
+    if dbnameInput != '':
         dbname = dbnameInput
-    if (userInput!=''):
+    if userInput != '':
         user = userInput
-    if (hostInput!=''):
+    if hostInput != '':
         host = hostInput
 
     try:
-        conn = psycopg2.connect("dbname="+dbname+" user="+user+" host="+host+" password="+password)
+        conn = psycopg2.connect("dbname=" + dbname + " user=" + user + " host=" + host + " password=" + password)
         print("Successfully connect to database!")
         connectStatus = True
         conn.autocommit = True
     except:
         print("Fail to connect to the database, please try again!")
 
-if (connectStatus):
+if connectStatus:
     cur = conn.cursor()
-    while(True):
-        print("Select your sql file (sql contains no comment!): ")
-        query_path = easygui.fileopenbox()
-        file_name = query_path.split("\\")[-1].split(".")[0]
+    while True:
+        print("\n*** Select your sql file (sql contains no comment!): ")
+        query_path = easygui.fileopenbox(default=os.path.join('..', 'data', 'sql', "*"))
+        file_name = query_path.split(os.path.sep)[-1].split(".")[0]
+        print("Your sql file path: ", query_path)
         query = open(query_path, "r", encoding="utf8").read()
-        print("\nShow me the databases:\n")
 
+        # print("\nShow me the databases:\n")
         # command = query.split(' ')[0]
-        # if (command in {'SELECT', 'INSERT', 'UPDATE', 'DELETE', 'VALUES', 'EXECUTE', 'DECLARE'}):
+        # if command in ['SELECT', 'INSERT', 'UPDATE', 'DELETE', 'VALUES', 'EXECUTE', 'DECLARE']:
         #     query_qep = 'EXPLAIN (FORMAT JSON) ' + query
-        # elif (command == 'CREATE' and query.split(' ')[1] == 'TABLE'):
+        # elif command == 'CREATE' and query.split(' ')[1] == 'TABLE':
         #     query_qep = 'EXPLAIN (FORMAT JSON) ' + query
         # print(query_qep)
 
         query_array = query.split(';')
```

Comment:
query_array contains all queries in .sql file, which are separeted by semicolon (:)

Reply:
cái này same mà đúng ko? có phải sửa j ở đây ko?